modules/ossm-member-roll.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/installing-ossm.adoc
+
+[id="ossm-member-roll_{context}"]
+= Configure the {ProductName} member roll
+
+You must create a `ServiceMeshMemberRoll` resource named `default` associated with the {ProductShortName} in the same project as the `ServiceMeshControlPlane`.
+
+[WARNING]
+====
+If Container Network Interface (CNI) plugin is enabled, manual sidecar injection will work, but pods will not be able to communicate with the control plane unless those pods are specified in the `ServiceMeshMemberRoll` resource.
+====
+
+[NOTE]
+====
+The member projects are only updated if the {ProductShortName} control plane installation succeeds.
+====
+
+* You can add any number of projects, but a project can only belong to *one* `ServiceMeshMemberRoll`.
+
+The `ServiceMeshMemberRoll` is deleted when its corresponding `ServiceMeshControlPlane` resource is deleted.
+
+[id="ossm-member-roll-console_{context}"]
+== Configure the member roll from the {product-title} web console
+
+Follow this procedure to add the Bookinfo project to the {ProductShortName} member roll by using the web console.
+
+.Prerequisites
+* An installed, verified {ProductName} Operator.
+
+.Procedure
+
+. Log into the {product-title} web console.
+
+. Click to *Home* -> *Projects*.
+
+. Click *Create Project*.
+
+. Enter a *Project Name* (for example, *bookinfo*), a *Display Name*, and a *Description*, then click *Create*.
+
+. Click *Catalog* -> *Installed Operators*.
+
+. Click the *Project* menu and choose `openshift-operators` from the list.
```

Comment:
openshift-operators namespace is incorrect.  Should be istio-system instead.

Reply:
I changed this, to istio-system, but if I need to change it back, let me know @travisrogers05.